PythonVisualizations/VisualizationApp.py

```diff
@@ -67,23 +49,51 @@ def gridDict(frame):
              for s in slaves],
             slaves))
 
+
 # Utilities for validating characters typed in Tk entry widgets
 # These must be registered with the parent window before use
 def numericValidate(action, index, value_if_allowed,
-             prior_value, text, validation_type, trigger_type, widget_name):
+                    prior_value, text, validation_type, trigger_type, widget_name):
     return len(value_if_allowed) == 0 or value_if_allowed.isdigit()
 
+
 geom_delims = re.compile(r'[\sXx+-]')
 
-class VisualizationApp(object): # Base class for Python visualizations
-    def __init__(            # Constructor
-            self, 
-            window=None,     # Run visualization within given window
+
+class VisualizationApp(object):  # Base class for Python visualizations
+
+    # Default styles for display of values and operational controls
+    FONT_SIZE = 20
+    VALUE_FONT = ('Helvetica', FONT_SIZE)
+    VALUE_COLOR = 'black'
+    _FONT = ('Courier', FONT_SIZE * 8 // 10)
+    VARIABLE_COLOR = 'brown3'
+    FOUND_FONT = ('Helvetica', FONT_SIZE)
+    FOUND_VARIABLECOLOR = 'green2'
```

Comment:
I think this was meant to be `FOUND_COLOR`. That is what it was defined as before